index.js

```diff
@@ -47,6 +47,7 @@ module.exports = function(opts) {
     if (user || opts.passthrough) {
       this.state = this.state || {};
       this.state[opts.key] = user;
+      this.state.token = token;
```

Comment:
I like the idea of this, however I'm sure not everyone would want to use the same state key as they may have something else under token. If instead this was made configurable it would be even better:

```
this.state[opts.tokenKey] = token;
```